csharp/src/Ice/Internal/MetricsAdminI.cs

```diff
@@ -690,17 +690,17 @@ public static void validateProperties(string prefix, Ice.Properties properties)
             }
         }
 
-        if (unknownProps.Count != 0 && properties.getIcePropertyAsInt("Ice.Warn.UnknownProperties") > 0)
+        if (unknownProps.Count != 0)
         {
-            StringBuilder message = new StringBuilder("found unknown IceMX properties for `");
+            StringBuilder message = new StringBuilder("Found unknown IceMX properties for '");
             message.Append(prefix.AsSpan(0, prefix.Length - 1));
             message.Append("':");
             foreach (string p in unknownProps)
             {
                 message.Append("\n    ");
                 message.Append(p);
             }
-            Ice.Util.getProcessLogger().warning(message.ToString());
+            throw new InvalidOperationException(message.ToString());
```

Comment:
Not clear what is the more appropriate exception here, this can be called from the dispatch when metrics are updated using the metrics admin facet.

Reply:
It should be a LocalException since we have a lot of code that expects only LocalException.